### DISCUSSION TITLE
Running an Epyx CRT file for C64OS Loading.

### DISCUSSION THREAD

**Discussion by yoalieh** [frntc/RAD#29] [Ideas]
I've been keeping an eye on C64Os, and as far as I understand, I see it can use 17xx REU, and it runs from an SD card running in an sdiec, thus it is slow to run.

I was thinking if the RAD could be able to run an Epyx CRT to load C64Os faster from the SDIEC, while taking advantage of the 17xxx feature of the RAD.

**Comment by frntc** [comment]
Hey, I've never tried C64OS. Does it not bring it's own floppy speeder?

At the moment the RAD cannot emulate CRTs (the hardware is not designed for it, maybe there will be some workaround in the future). In case that C64OS uses kernal disk routines, you could try JiffyDOS!?

**Reply by yoalieh** [reply]
I think that's another option mentioned on the c64os webpage. I think I'll try fetching a jiffydos to Mexico. Greetings!

**Reply by yoalieh** [reply]
Sorry, I didn't answered your question:

No, it doesn't includes any speeder, it has been designed for sdiec usage. It can benefit from JiffyDos, or other 1541 speeders.

As the c64os author has written in his page, it will take advantage of 17xx ram starting from 1.05 version to send apps to the background, and allow fast switching between them.

I'll start by getting a RAD REU kit, flashing a licensed jiffy dos rom, and soon I hope I can get the c64os sd card.

I'm impressed with your projects btw :-)
